Cinema/src/main/java/com/example/Cinema/validator/OrderForCheck.java

```diff
@@ -1,7 +1,8 @@
 package com.example.Cinema.validator;
 
 import javax.validation.GroupSequence;
+import javax.validation.groups.Default;
 
-@GroupSequence({FirstPriorInfo.class, SecondPriorInfo.class, ThirdPriorInfo.class})
+@GroupSequence({Default.class, FirstPriorInfo.class, SecondPriorInfo.class, ThirdPriorInfo.class})
 public interface OrderForCheck {
```

Comment:
It might be a good idea to rename this into RequestValidationSequence, or something like that, to make it clearer what this interface is for.

Reply:
Renamed